api/hooks/commit-msg

```diff
@@ -2,6 +2,39 @@
 
 echo -e "Checking commit message format"
 MSG_FILE=$1
+
+INPUT_MSG=$(cat $MSG_FILE)
+
+HAVE_PREFIX=$(echo $INPUT_MSG | grep -E '.*(PC-[0-9]+).*')
+if [ -z "$HAVE_PREFIX" ]; then
+    CURRENT_BRANCH=$(git br --show-current)
+    JIRA_ID=$(echo $CURRENT_BRANCH | sed -E 's/.*(PC-[0-9]+).*/\1/')
+    HAVE_CHANGE_ADAGE=`git diff --cached --name-only | grep ^adage-front/`
+    HAVE_CHANGE_API=`git diff --cached --name-only | grep ^api/`
+    HAVE_CHANGE_PRO=`git diff --cached --name-only | grep ^pro/`
+
+    REPOSITORY_SCOOP_LIST=()
```

Comment:
```suggestion
    REPOSITORY_SCOPE_LIST=()
```

ou juste `SCOPES` ?

Reply:
I use to name variables `toto` and `toto_list` instead of `toto` and `totos`
Here it's no `REPOSITORY_SCOPE` but i named it `_LIST` out of habit.